### ISSUE TITLE
Error when executing program

### ISSUE THREAD

**Issue by HandyGuySoftware** [HandyGuySoftware/dupReport#123]
```
Traceback (most recent call last):
  File "/home/steve/dupReport/dupReport.py", line 274, in <module>
    msgHtml, msgText, msgCsv = rpt.runReport(startTime)
  File "/home/steve/dupReport/rpt_bydest.py", line 172, in runReport
    msgHtml, msgText, msgCsv = report.rptBottom(msgHtml, msgText, msgCsv, startTime, nFields)
  File "/home/steve/dupReport/report.py", line 209, in rptBottom
    msgHtml2, msgText2, msgCsv2 =  lastSeenTable(globs.report.reportOpts)
  File "/home/steve/dupReport/report.py", line 363, in lastSeenTable
    dVersion = dbCursor.fetchone()[0]
TypeError: 'NoneType' object is not subscriptable
```

**Comment by HandyGuySoftware** [comment]
Occurs when trying to fetch the Duplicati version number from an email record where it isn't present. Earlier versions of Duplicati did not return the version number in the email. Need to account for that.

**Comment by HandyGuySoftware** [comment]
Not sure why this wasn't crashing the program before, but fixed in 2.2.8.
